lab2.ipynb

```diff
@@ -779,6 +583,185 @@
+    "ax.plot(x, scipy.stats.norm.pdf(x)*10000,\n",
+    "       'r-', lw=5, alpha=0.6, label='norm pdf')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e985d33f-c633-4b86-9d08-116fd681260e",
+   "metadata": {},
+   "source": [
+    "The above graph shows the probability distribution of 10k pixels, with a 0 mean gaussian that has a width of 2. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b1f8a119-b88b-4020-8e29-7933bd410d2a",
+   "metadata": {},
+   "source": [
+    "## B) Significance of meassurement"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "42b591e6-1101-4299-9b6b-7300123264db",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scipy.stats.norm.isf(probability*10000)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "843bb1f6-3e80-459c-83e2-d889578f37bd",
+   "metadata": {},
+   "source": [
+    "Here, I multiplied the probability by the number of events that occured to see the total probability of meassuring this value or higher. Then, I took the isf of it to find the sigma value of the probability."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8a373914-5312-4dc2-aabb-e03840a22b38",
+   "metadata": {},
+   "source": [
+    "# Problem 4"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cc833f63-b533-4055-8670-5df01da5031d",
+   "metadata": {},
+   "source": [
+    "## A) 5-sigma detection in version 1"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c74d0fe7-14fa-4122-bffa-fc18e9d6d4a4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sigma_5 = scipy.stats.norm.sf(5)\n",
+    "scipy.stats.norm.isf(sigma_5, scale = x)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8517b54b-db47-45d2-a4d8-1210a0d07d74",
+   "metadata": {},
+   "source": [
+    "A 5 sigma distiribution would have a probability of $2.8 \\times 10^{-7}$, which when the probability distribution has a width of 2, we see that the corrosponding value needed to achive 5 sigma is 10. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "579f3472-8151-47ab-a0e3-e19c9d7dfcc2",
+   "metadata": {},
+   "source": [
+    "## B) 5-sigma detection in version 2"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "35bf5deb-9d84-4ed4-baa4-2804fb18882c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scipy.stats.norm.isf(1/(3500000*10000), scale = 2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7469b8b8-ef73-42ae-ae63-bcb11363b3a5",
+   "metadata": {},
+   "source": [
+    "For a sigma event to occur, we would expect a signal of at least 13.1 to occur given our background distribution. This is higher than the previous 5-sigma event, since there is a chance to measure a 5-sigma event simply from the backgroun distribution itself. Taking this into account, we see that we now have a higher bar to clear in order to meassure an event of significance."
+   ]
```

Comment:
Hey I am not sure if you forgot to push, but you are missing part C when we are asked to talk about how much brighter our signal be at 10k pixels. Done by just taking the ratio for a signal corresponding to 5 sigmas for more in the 1k pixels to the 5 sigma signal here. 

We were also asked to change the trials factor to something much bigger and compare this ratio.